SubmittingPatches-backports.rst

```diff
@@ -326,6 +326,12 @@ editing. If you need to add additional information to the cherry-pick commit
 message, append that information below this line. Once again: do not modify the
 original commit message.
 
+In order to get the PR created by the script, go back to master to run the same
```

Comment:
@Devp00l I'm confused by "go back to master" here. Were you running `src/script/ceph-backport.sh` directly from the nautilus or octopus code tree?

Reply:
Ah, now I see the possible confusion. You are starting on "master" and running `src/script/ceph_backport.sh`. Then when there is a conflict, you are suddenly in a branch (octopus, master) with an old, deprecated version of the script.

The solution (which is documented already), is to never run `src/script/ceph_backport.sh`. Always put the script somewhere in your PATH before running it.

Reply:
Added the word `copy` somewhere in this documentation ;)